src/popup/components/Modals/AccountCreate.vue

```diff
@@ -47,12 +59,20 @@ export default {
     loading: false,
   }),
   methods: {
+    ...mapActions('modals', {
```

Comment:
in this pr, i create useDispatch fot that, https://github.com/aeternity/superhero-wallet/pull/1784

Reply:
You `useDispatch` can only be used within the `setup()` hook of Composition API components. So this can be resolved.